sktime/utils/plotting.py

```diff
@@ -504,3 +504,66 @@ def get_y(length, split):
         return fig, ax
     else:
         return ax
+
+
+def plot_calibration(y_true, y_pred, ax=None):
+    """Plot the calibration of a probabilistic forecast.
+
+    Calculates internally the calibration of the quantile forecast and
+    visualise it.
+
+    x-axis: interval from 0 to 1
+    y-axis: interval from 0 to 1
+    plot elements: the calibration fo the forecast (blue) and the ideal
+        calibration (orange)
+
+    Parameters
+    ----------
+    y_true : pd.Series
```

Comment:
could also be single-series iterable, e.g., 1-column `pd.DataFrame` etc?

Reply:
Yes also possible are single columned pd.DataFrames and np.arrays

Reply:
so the docstrig should say so, no?